cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/ReportService.java

```diff
@@ -47,18 +51,29 @@ public ReportService() {
     @GET
     @Path("/parameters/{report_type}")
     @Produces(MediaType.APPLICATION_JSON)
-    public Response getReportParameters(@PathParam("report_type") String reportType) {
+    public Response getReportParameters(@PathParam("report_type") String reportTypeStr) {
         try {
-            if (reportType == null || ReportType.valueOf(reportType.toUpperCase()) == null) {
-                throw new RuntimeException("Report type is unknown.");
-            }
+            Objects.requireNonNull(reportTypeStr, "Report type is unknown.");
 
-            ReportParameters parameters = ReportType.valueOf(reportType.toUpperCase()).getParameters();
+            ReportType reportType = ReportType.valueOf(reportTypeStr.toUpperCase());
+            Objects.requireNonNull(reportType, "Report type is unknown.");
```

Comment:
Even when reportTypeStr can be nullable reportType can't be nullable because Enum#valueOf method throws NPE in canse when enum name paremeter is nullable